tst/children.tst

```diff
@@ -0,0 +1,29 @@
+gap> d := DirectoryCurrent();;
```

Comment:
This overwrites the `d` variable from `testall.g`, leading to the test failing. Rename to e.g. `dir`?

Reply:
Of course we wouldn't have this issue if we used `TestDirectory`. I'll look into that as part of PR #52, then you can rebase on that.